pkg/proxy/server.go

```diff
@@ -90,18 +92,25 @@ func newProxyServer(
 	}
 
 	return &ProxyServer{
-		kubeClient:       client,
-		istioClient:      istioClient,
-		IptInterface:     iptInterface,
-		execer:           execer,
-		Proxier:          proxier,
-		ConfigSyncPeriod: config.ConfigSyncPeriod.Duration,
-		loadBalancer:     loadBalancer,
+		kubeClient:        client,
+		istioClient:       istioClient,
+		IptInterface:      iptInterface,
+		execer:            execer,
+		Proxier:           proxier,
+		ConfigSyncPeriod:  config.ConfigSyncPeriod.Duration,
+		loadBalancer:      loadBalancer,
+		serviceFilterMode: serviceFilterMode,
 	}, nil
 }
 
 func (s *ProxyServer) Run() error {
-	noEdgeMeshProxyName, err := labels.NewRequirement(defaults.LabelEdgeMeshServiceProxyName, selection.DoesNotExist, nil)
+	// Determine the service filter mode.
+	// By default, we will proxy all services that are not labeled with the LabelEdgeMeshServiceProxyName label.
+	operation := selection.DoesNotExist
+	if s.serviceFilterMode != defaults.DefaultServiceFilterMode {
```

Comment:
Don't use DefaultServiceFilterMode, users don't know what it means. Use defaults.FilterIfLabelExistsMode

Reply:
done. used first one.